src/components/editMeasure/editor/valuesetApplier.ts

```diff
@@ -122,61 +144,39 @@ const getValueSetTitleName = (vs) => {
   return vs.title;
 };
 
-//this function is to check if the values set is being added or updated
-const checkIfValueSetBeingEdited = (vs, previousVs) => {
-  if (
-    previousVs &&
-    previousVs.title === vs.title &&
-    previousVs.steward === vs.steward &&
-    previousVs.oid === vs.oid
-  ) {
-    return true;
-  }
-  return false;
-};
-
-// this to remove the previous line that was added when updating the valueset
-const updateCql = (cql, previousVs) => {
-  const valueSetStatement = `valueset "${getValueSetTitleName(previousVs)}": '${
-    previousVs.oid
-  }'`;
-  const normalize = (str) => str.replace(/\s+/g, " ").trim();
-  const normalizedText = normalize(valueSetStatement);
-  const updatedCqlArray = cql.split("\n").filter((statement) => {
-    const normalizedItem = normalize(statement);
-    return normalizedItem.localeCompare(normalizedText) !== 0;
-  });
-
-  return updatedCqlArray.join("\n");
-};
-
 const applyValueset = (
   cql: string,
   vs: ValueSetForSearch,
   previousVs?: ValueSetForSearch
 ): CodeChangeResult => {
-  const updatedCql = checkIfValueSetBeingEdited(vs, previousVs)
-    ? updateCql(cql, previousVs)
-    : cql;
-  const cqlArr: string[] = updatedCql.split("\n");
-  const parseResults: CqlResult = new CqlAntlr(updatedCql).parse();
+  const cqlArr: string[] = cql.split("\n");
+  const parseResults: CqlResult = new CqlAntlr(cql).parse();
   let valuesetChangeStatus: "success" | "info" | "danger" = "danger";
   let message: string = "The requested operation was unsuccessful";
```

Comment:
value not being used, can be removed I think 
```suggestion
  let message: string;
```